kbase-extension/static/kbase/js/widgets/narrative_core/kbaseCellToolbarMenu.js

```diff
@@ -1,24 +1,23 @@
 define([
     'jquery',
-    'kb_common/html',
+    'common/html',
     'common/events',
     'base/js/namespace',
     'common/utils',
-    'util/bootstrapDialog',
-    'kbase/js/widgets/appInfoPanel',
     'narrativeConfig',
+    'common/jobs',
```

Comment:
Ran eslint on the file, removed quite a bit of cruft, and then refactored the cell toolbar to remove a load of unneeded DIVs.